.rubocop.yml

```diff
@@ -85,6 +85,9 @@ Metrics/LineLength:
 Metrics/ModuleLength:
   Severity: refactor
 
+Metrics/BlockLength:
+  Exclude:
+    - 'src/api/config/routes.rb'
```

Comment:
at the same time you exclude this, you should regenerate the todo, as the blocklength may change. However this is also done in https://github.com/openSUSE/open-build-service/pull/4842. So if the other one is merged before, you can just remove this and rebase :wink:

Reply:
@mdeniz #4842 is merged! Remove this and rebase! :wink:

Reply:
Done, please re-review it :+1: